td.vue/package.json

```diff
@@ -5,6 +5,8 @@
   "appBundleId": "org.owasp.threatdragon",
   "private": true,
   "scripts": {
+    "prebuild:desktop": "node ./src/plugins/generate-cornucopia.js",
```

Comment:
How are these scripts intended to be run? If manually then we need to update `docs`, if during the build process then probably add them to `build` script below